<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stackedwidget</class>
 <widget class="QWidget" name="stackedwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stackedwidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="stackedwidget_widget_1" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <widget class="QComboBox" name="stackedwidget_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="stackedwidget_pushButton_2">
      <property name="geometry">
       <rect>
        <x>278</x>
        <y>10</y>
        <width>103</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>show awidget</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stackedwidget_pushButton_3">
      <property name="geometry">
       <rect>
        <x>387</x>
        <y>10</y>
        <width>102</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>del awidget</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stackedwidget_pushButton_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>102</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add awidget</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stackedwidget_pushButton_4">
      <property name="geometry">
       <rect>
        <x>496</x>
        <y>10</y>
        <width>102</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>show index 0</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedwidget_stackedWidget">
     <widget class="QWidget" name="stackedwidget_page_1"/>
     <widget class="QWidget" name="stackedwidget_page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
